testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -54,6 +54,8 @@
 
     SUPPORTS_DELETE(false),
     SUPPORTS_ROW_LEVEL_DELETE(SUPPORTS_DELETE),
+    // See io.trino.spi.connector.ConnectorMetadata.supportsMetadataDelete
+    SUPPORTS_METADATA_DELETE(SUPPORTS_DELETE),
```

Comment:
Sadly, this is a wrong name. It's hive-ism. "metadata delete" doesn't make any sense for JDBC connectors and that's _not_ what implemented.

What we're looking for here is "delete delegation" (or "delete 'pushdown'", but i think delegation is a better term).

(Fortunately, there is nothing in testing connector behavior that requires us to reuse naming from SPI, and we already leverage that.)